tests/test_tekton.py

```diff
@@ -820,20 +821,25 @@ def test_wait_for_start(self, pipeline_run):
         responses.add(responses.GET, PIPELINE_RUN_URL, json=PIPELINE_RUN_JSON)
         resp = pipeline_run.wait_for_start()
 
-        assert len(responses.calls) == 2
```

Comment:
Why the changes in this test? There were no changes in the method

(I'm guessing the `time.sleep()` mocking is to make things faster, but what about this one?)